Makefile.toml

```diff
@@ -137,6 +140,7 @@ dependencies = [
 
 [tasks.ci-job-nostd]
```

Comment:
Can you also fix ci-job-msrv-features-1, ci-job-msrv-features-2, ci-job-msrv-features-3?